### ISSUE TITLE
Support for new Dart language version by fixing nullability issues

### ISSUE THREAD

**Issue by thalescm** [Salby/morph#22]
Currently this library cannot be used on newer versions of Dart due to null safety improvements:
```
Error: Cannot run with sound null safety, because the following dependencies
don't support null safety:

- package:morpheus
2

For solutions, see https://dart.dev/go/unsound-null-safety
```

**Comment by Salby** [comment]
Thanks for opening an issue. I'll take a look at this and have it fixed ASAP.

**Comment by Salby** [comment]
Fixed in version 1.2.3
